website/www/site/content/en/get-started/downloads.md

```diff
@@ -87,10 +87,17 @@ versions denoted `0.x.y`.
 
 ## Releases
 
+### 2.22.0 (2020-06-09)
```

Comment:
Why tomorrow? (Though it doesn't really matter)

Reply:
So I was actually trying to replicate what you did. You sent out the release vote result on 5/26 promising to announce the release on 5/27, and the latter is the date referenced everywhere.

_however_ - it looks like the reason for that is you didn't actually merge the PRs until 5/27 (https://github.com/apache/beam/pull/11727). I'll set the dates back to 6/8 here

Reply:
Yeah it was the evening and I was sleepy haha :D